tests/php/bootstrap.php

```
@@ -21,6 +21,19 @@
     define('ABSPATH', dirname(__DIR__, 2) . '/');
 }
 
+// Mock WordPress core functions.
+if ( ! function_exists( 'get_locale' ) ) {
+	function get_locale() {
+		return 'en_US';
+	}
+}
+
+if ( ! function_exists( 'apply_filters' ) ) {
+	function apply_filters( $_hook_name, $value, ...$args ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$_hook_name'. (undefined)

(UnusedFormalParameter)